e2e/setup/config.go

```diff
@@ -80,15 +81,18 @@ func getConfig(config *Config, path string) error {
 
 // Setup used by the e2e tests. Preloaded with all necessary dependencies
 type Setup struct {
-	Receiver      common.Address
-	BridgeAccount hederaSDK.AccountID
-	SenderAccount hederaSDK.AccountID
-	TopicID       hederaSDK.TopicID
-	TokenID       hederaSDK.TokenID
-	FeePercentage int64
-	Members       []hederaSDK.AccountID
-	Clients       *clients
-	DbValidation  *db_validation.Service
+	BridgeAccount             hederaSDK.AccountID
+	HederaSender              hederaSDK.AccountID
+	HederaReceiver            hederaSDK.AccountID
```

Comment:
I think it would be best to not add these, but use `Clients.Hedera.GetOperatorAccountID()`, so that the bridge transfers rotate from one place to the same place. This way it would be easier to handle the configuration and not having to fund more than 1 account ID (`hedera.sender.account`)

Reply:
Example: `0.0.1234` bridges to `0x7890`, then `0x7890` returns back to `0.0.1234`